python/tvm/relay/op/transform.py

```diff
@@ -613,13 +613,13 @@ def strided_slice(data, begin, end, strides=None):
     data : relay.Expr
         The source array to be sliced.
 
-    begin: list of int
+    begin: relay.Expr
```

Comment:
Should we allow begin, end and strides to be list/tuple as well? And do convertsion to const before calling backend API.